cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -66,6 +66,10 @@ type MasterConfiguration struct {
 	Token string
 	// TokenTTL is a ttl for Token. Defaults to 24h.
 	TokenTTL *metav1.Duration
+	// Describes the ways in which this token can be used.
+	TokenUsages []string
```

Comment:
Follow [golang convention for exported types](https://golang.org/doc/effective_go.html#commentary) by changing this to `TokenUsages describes the ways...`. Similar for elsewhere in this PR.